### PR TITLE
79 sign your report

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: test/views/SignAndDateReportWebTest.java [hunk 1]
```diff
@@ -5,38 +5,95 @@
...
+        return String.format("{\"templateName\": \"fooBar\", \"values\": { \"pageNumber\": \"11\", \"name\": \"Smith,John\", \"address\": \"1234\", \"pnc\": \"Retrieved From Store\",  \"startDate\": \"%s\" } }", startDate);
     }
 
     @Override
     protected Application provideApplication() {
         return new GuiceApplicationBuilder().
             overrides(
                 bind(PdfGenerator.class).toInstance(new SimplePdfGeneratorMock()),
-                bind(DocumentStore.class).toInstance(new SimpleDocumentStoreMock()),
+                bind(DocumentStore.class).toInstance(new SimpleDocumentStoreMock() {
```
Comment (retroredge) [inline test/views/SignAndDateReportWebTest.java]: Change for a lambda

File: app/views/shortFormatPreSentenceReport/page8.scala.html [hunk 2]
```diff
@@ -43,6 +43,7 @@ <h1 class="heading-xlarge">Risk assessment</h1>
             @inputHidden(reportForm("onBehalfOfUser"), encrypter)
             @inputHidden(reportForm("entityId"), encrypter)
             @inputHidden(reportForm("documentId"), encrypter)
+            @inputHidden(reportForm("startDate"), encrypter)
```
Comment (retroredge) [inline app/views/shortFormatPreSentenceReport/page8.scala.html]: Move to page 11 (repeat all pages)

File: app/controllers/base/ReportGeneratorWizardController.java [hunk 3]
```diff
@@ -127,7 +130,7 @@ protected final String baseViewName() {
                     return ImmutableMap.of("errorMessage", ThrowableHelper.toMessageCauseStack(error));
                 }).
                 thenApply(stored -> {
-
+                    params.put("startDate", LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")));
```
Comment (retroredge) [inline app/controllers/base/ReportGeneratorWizardController.java]: Move to data bean dto

File: app/views/shortFormatPreSentenceReport/page8.scala.html (hunk 2 above)
Comment (andymarke) [reply app/views/shortFormatPreSentenceReport/page8.scala.html]: Done. We also missd adding

        @inputHidden(reportForm("courtOfficePhoneNumber"))
        @inputHidden(reportForm("counterSignature"))

to all pages

**Pull request merged**